landscape_analysis.md

```diff
@@ -10,20 +10,28 @@ this proposal interact? (maximum of 250 words)
 
 <!-- 243/250 words -->
 
-Nibabel is the base layer for the Nipy (Neuroimaging in Python) community of projects.  It is the basic workbench and input/output layer on which higher-level libraries can build.
+Nibabel is the base layer for the Nipy (Neuroimaging in Python) community of projects.  It is the basic workbench and input/output (I/O) layer on which higher-level libraries can build.
 
 Neuroimaging researchers need flexible tooling to analyze neural data. Python
 and Numpy provide great power and flexibility, and Nibabel provides data
 structures and common algorithms to apply Python libraries to imaging data.
 
-Researchers using Python for neuroimaging are all but invariably using Nibabel either directly, or via a high-level library.
+Researchers using Python for neuroimaging are all but invariably using Nibabel either directly, or via a high-level library (e.g., Nilearn or DIPY).
 
-The most direct competitor to the Python/Nibabel ecosystem is SPM,
```

Comment:
This phrasing was deliberate - because there is a fundamental difference between programming language / API + tools on top (Nibabel etc, SPM), and tools with some scripting (FSL, AFNI etc).

Reply:
I think splitting the world in MATLAB/SPM vs others is not very effective and misrepresents the landscape about which they are asking (that w.r.t. the difference you want to point out). Also, in my suggestion I identify the pattern you propose, and indicate explicitly such a model. In other words, I don't see the difference you are making between Nibabel and SPM and then FSL or AFNI.

Regarding the particular term "competitor", I think we should sound integrative. IMHO "alternatives" is better.